scripts/EN/Harry

```diff
@@ -3355,9 +3355,9 @@ CleanScreen(); // 0x40d2: 00000064
 LoadBackground(0x54); // 0x40d6: 00001069
 DrawBackground(); // 0x40dc: 00000065
 PlayMusic(0x6e, 0x1); // 0x40e0: 0000206c
-ShowText((1291)"I...get out the pet bottle and use a fragment of the bottom section of one of the broken bottles to pour the vermilion liquid into the it."); // 0x40e8: 0000100c
+ShowText((1291)"I…get out the plastic bottle and use a fragment of the bottom section of one of the broken vials to pour the vermilion liquid into the it."); // 0x40e8: 0000100c
 HandleInput(); // 0x40ee: 00000010
-ShowText((1292)"The sticky liquid drips into the pet bottle. The last drop lingered, but once it finally hit the surface of the liquid already inside the pet bottle I tightly closed the cap and then left the Director's Office."); // 0x40f2: 0000100c
+ShowText((1292)"The sticky liquid drips into the plastic bottle.¥nW2The last drop lingered, W1but once it finally hit the surface of the liquid already inside the plastic bottle W1I tightly closed the cap and then left the Director's Office."); // 0x40f2: 0000100c
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)